src/System.Net.WebSockets.Client/tests/System.Net.WebSockets.Client.Tests.csproj

```diff
@@ -40,5 +53,8 @@
     <Compile Include="WebSocketData.cs" />
     <Compile Include="WebSocketHelper.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <SupplementalTestData Include="$(PackagesDir)system.net.testdata\1.0.0-prerelease\content\**\*.*" />
+  </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
 </Project>
```

Comment:
Weird....Visual Studio must have undone my newline fix :( .  Trying again in the next iteration....